config/feeds_config_v2.json

```diff
@@ -53660,7 +53660,7 @@
       }
     },
     {
-      "id": 1000003,
+      "id": 2000000,
```

Comment:
maybe change this value to all other places

Reply:
The main idea is to organize feeds into distinct logical groups and leave free ID slots for potential future additions.